localstack/services/awslambda/invocation/executor_endpoint.py

```diff
@@ -124,7 +124,9 @@ def invoke(self, payload: Dict[str, str]) -> None:
         if not self.container_address:
             raise ValueError("Container address not set, but got an invoke.")
         invocation_url = f"http://{self.container_address}:{self.container_port}/invoke"
-        response = requests.post(url=invocation_url, json=payload)
+        # disable proxies for internal requests
+        proxies = {"http": "", "https": ""}
+        response = requests.post(url=invocation_url, json=payload, proxies=proxies)
```

Comment:
We should in the future maybe write a small wrapper and make sure we're using the wrapper for internal requests. :thinking: